fenicsxprecice/adapter_core.py

```diff
@@ -164,7 +164,7 @@ def get_fenicsx_vertices(function_space, coupling_subdomain, dims):
         else:
             coords = np.array([])
     except Exception as e:  # fall back to old method  # TODO is that too general? Better use, e.g., IndexError here?
-        print("Caught the following exception in the detection of the coupling subdomain:\n{e}")
+        print(f"Caught the following exception in the detection of the coupling subdomain:\n{e}")
```

Comment:
Lets add changes which are not directly related to the topic of the pull request directly on the develop branch (via a commit).